main.nf

```diff
@@ -24,196 +24,226 @@ def helpMessage() {
+      --maltextract_matches [bool]              Turn on exporting alignments of hits in BLAST format.
+      --maltextract_megansummary [bool]         Turn on export of MEGAN summary files.
+      --maltextract_percentidentity [num]       Minimum percent identity alignments are required to have to be reported. Recommended to set same as MALT parameter. Default: ${params.maltextract_percentidentity}
+      --maltextract_topalignment [int]          Turn on using top alignments per read after filtering.
 
     Other options:     
-      -name                         Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic.
-      --max_memory                  Memory limit for each step of pipeline. Should be in form e.g. --max_memory '8.GB'. Default: '${params.max_memory}'
-      --max_time                    Time limit for each step of the pipeline. Should be in form e.g. --max_memory '2.h'. Default: '${params.max_time}'
-      --max_cpus                    Maximum number of CPUs to use for each step of the pipeline. Should be in form e.g. Default: '${params.max_cpus}'
-      --email                       Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits
-      --plaintext_email             Receive plain text emails rather than HTML
-      --maxMultiqcEmailFileSize     Threshold size for MultiQC report to be attached in notification email. If file generated by pipeline exceeds the threshold, it will not be attached (Default: 25MB)
+      -name [str]                     Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic.
+      --max_memory [str]              Memory limit for each step of pipeline. Should be in form e.g. --max_memory '8.GB'. Default: '${params.max_memory}'
+      --max_time [str]                Time limit for each step of the pipeline. Should be in form e.g. --max_time '2.h'. Default: '${params.max_time}'
+      --max_cpus [str]                Maximum number of CPUs to use for each step of the pipeline. Should be in form e.g. Default: '${params.max_cpus}'
+      --email [email]                 Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits
+      --email_on_fail [email]         Same as --email, except only send mail if the workflow is not successful
+      --plaintext_email [email]       Receive plain text emails rather than HTML
+      --max_multiqc_email_size [str]  Threshold size for MultiQC report to be attached in notification email. If file generated by pipeline exceeds the threshold, it will not be attached (Default: 25MB)
+
+    AWSBatch options:
+      --awsqueue [str]   The AWSBatch JobQueue that needs to be set when running on AWSBatch
+      --awsregion [str]  The AWS Region for your AWS Batch job to run on
+      --awscli [str]     Path to the AWS CLI tool
```

Comment:
I skipped the help message, I assume they have the same text as the nextflow_schema.json and I will check them there.

Reply:
Yes, they are (read: should be) the same, but good you commented exactly here: I actually forgot the AWS ones in the schema as they were accidently deleted in config :laughing:.